yark/viewer.py

```diff
@@ -103,7 +103,15 @@ def video(name, kind, id):
     try:
         # Get information
         archive = Archive.load(name)
-        video = archive.search(id)
+        if kind == "videos":
+            video = archive.search_videos(id)
+        elif kind == "livestreams":
+            video = archive.search_livestreams(id)
+        elif kind == "shorts":
+            video = archive.search_shorts(id)
+        else:
+            return
```

Comment:
This return isnt needed because of the if above, looks good apart from this :)

Sorry for not catching this the first time

Reply:
np, thank you